docs/reporters.md

```diff
@@ -0,0 +1,105 @@
+---
+title: InSpec Reporters
+---
+
+# InSpec Reporters
+
+Introduced in InSpec 1.51.6
+
+InSpec allows you to output your test results to one or more reporters. You can configure the reporter(s) using either the `--json-config` option or the `--reporter` option. While you can configure multiple reporters to write to different files, only one reporter can output to the screen(stdout).
+
+## Syntax
+
+You can specify one or more reporters using the `--reporter` cli flag. You can also specify a output by appending a path seperated by a colon.
+
+Output json to screen.
+
+```bash
+inspec exec --reporter json
+or
+inspec exec --reporter json:-
+```
+
+Output cli to screen and write json to a file.
+
+```bash
+inspec exec --reporter cli json:/tmp/output.json
+```
+
+Output nothing to screen and write junit and html to a file.
+
+```bash
+inspec exec --reporter junit:/tmp/junit.xml html:www/index.html
+```
+
+Output json to screen and write to a file. Write junit to a file.
+
+```bash
+inspec exec --reporter json junit:/tmp/junit.xml | tee out.json
+```
+
+If you are using the cli option `--json-config` you can also set reporters.
+
+Output cli to screen.
+
+```json
+{
+    "reporter": {
+        "cli" : {
+            "stdout" : true
+        }
+    }
+}
+```
+
+Output cli to screen and write json to a file.
+
+```json
+{
+    "reporter": {
+        "cli" : {
+            "stdout" : true
+        },
+        "json" : {
+            "file" : "/tmp/output.json",
+            "stdout" : false
+        }
+    }
+}
+```
+
+## Supported Reporters
+
+The following are the current supported reporters:
+
+### cli
```

Comment:
Can we change the order to:
- cli
- json
- json-min
- documentation
- junit
- progress
- json-rspec
- html